packages/callout-block/src/CalloutBlock.spec.ct.tsx

```diff
@@ -266,4 +266,22 @@ describe('Callout Block', () => {
         cy.get(CalloutBlockSelector).should('have.css', 'background-color', 'rgb(50, 40, 145)');
         cy.get(HtmlContentSelector).should('have.css', 'color', 'rgb(255, 255, 255)');
     });
+
+    it('renders a callout block with the content hugged', () => {
+        const [CalloutBlockWithStubs] = withAppBridgeBlockStubs(CalloutBlock, {
+            blockSettings: {
+                textValue: 'Some random text',
+                type: Type.Note,
+                appearance: Appearance.Strong,
+                width: Width.HugContents,
+            },
+        });
+
+        mount(<CalloutBlockWithStubs />);
+
+        cy.get(CalloutBlockSelector).should(($el) => {
+            const width = parseFloat($el.css('width'));
+            expect(width).to.be.greaterThan(100);
```

Comment:
can we do this more exactly as its also true if its full width what also would be wrong